packages/talos_cohort_replicator/src/services/statemap_installer_service.rs

```diff
@@ -11,13 +14,72 @@ pub struct StatemapInstallerConfig {
     pub thread_pool: Option<u16>,
 }
 
+async fn statemap_install_future(
```

Comment:
RE: method name with ending _future. What does it mean? When reading it is not straightforward to understand what will happen when method is called.

Reply:
Isn't `future` is pretty self explanatory in this context. 

To me it was pretty obvious based on being used in the tokio task, that this `fn` is the future expected in the tokio task.

Reply:
To me method name is about "do something" or "get something", "parse something", "create something" and etc From that context I don't know what _future  does here. Why do we need to encode a data type into the method name :)